### ISSUE TITLE
Integrate aXe library for accessibility coverage

### ISSUE THREAD

**Issue by suryapulakhandam** [nhsuk/nhsuk-frontend#383]
Integrate aXe library, so that we increase the accessibility coverage.

**Comment by suryapulakhandam** [comment]
Build is failing due some accessibility issues 

logs:

Violation of "aria-roles" with 3 occurrences!
    Ensures all elements with a role attribute use a valid value. Correct invalid elements at:
     - .nhsuk-care-card--primary > .nhsuk-care-card__heading-container > .nhsuk-care-card__heading > span[role="text"]
     - .nhsuk-care-card--urgent > .nhsuk-care-card__heading-container > .nhsuk-care-card__heading > span[role="text"]
     - .nhsuk-care-card--emergency > .nhsuk-care-card__heading-container > .nhsuk-care-card__heading > span[role="text"]
    For details, see: https://dequeuniversity.com/rules/axe/3.1/aria-roles
  Violation of "empty-heading" with 3 occurrences!
    Ensures headings have discernible text. Correct invalid elements at:
     - .nhsuk-care-card--primary > .nhsuk-care-card__heading-container > .nhsuk-care-card__heading
     - .nhsuk-care-card--urgent > .nhsuk-care-card__heading-container > .nhsuk-care-card__heading
     - .nhsuk-care-card--emergency > .nhsuk-care-card__heading-container > .nhsuk-care-card__heading
    For details, see: https://dequeuniversity.com/rules/axe/3.1/empty-heading
  Violation of "label" with 1 occurrences!
    Ensures every form element has a label. Correct invalid elements at:
     - #search-field
    For details, see: https://dequeuniversity.com/rules/axe/3.1/label
  Violation of "skip-link" with 22 occurrences!
    Ensure all skip links have a focusable target. Correct invalid elements at:
     - a[href="#E"]
     - a[href="#F"]
     - a[href="#G"]
     - a[href="#H"]
     - a[href="#I"]
     - a[href="#J"]
     - a[href="#K"]
     - a[href="#L"]
     - a[href="#M"]
     - a[href="#N"]
     - a[href="#O"]
     - a[href="#P"]
     - a[href="#Q"]
     - a[href="#R"]
     - a[href="#S"]
     - a[href="#T"]
     - a[href="#U"]
     - a[href="#V"]
     - a[href="#W"]
     - a[href="#X"]
     - a[href="#Y"]
     - a[href="#Z"]
    For details, see: https://dequeuniversity.com/rules/axe/3.1/skip-link
29 Accessibility issues detected.
